### PR TITLE
feat/API-1770-add-many-languages-snippets

### PR DESCRIPTION
**Pull request by damienSD**

### CODE REVIEW

**Comment by damienSD** [comment]
image

File: content/apps/create-app.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+First, your application must expose an activation URL.
+
+In our example, we won't do additional steps (like authentification), so we will launch the Authorization Request
+immediately in this Activation URL.
+
+
+!!!include(content/apps/create-app/activate-php.md)!!!
+!!!include(content/apps/create-app/activate-nodejs.md)!!!
+!!!include(content/apps/create-app/activate-python.md)!!!
+<!-- !!!include(content/apps/create-app/activate-java.md)!!!
```
Comment (tomglvng) [inline content/apps/create-app.md:32]: In my opinion, remove unused code: we will add it in another PR if necessary.

File: content/apps/create-app.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Callback URL
+
+Then, your application must expose a callback URL.
+
+
+!!!include(content/apps/create-app/callback-php.md)!!!
+!!!include(content/apps/create-app/callback-nodejs.md)!!!
+!!!include(content/apps/create-app/callback-python.md)!!!
+<!-- !!!include(content/apps/create-app/callback-java.md)!!!
```
Comment (tomglvng) [inline content/apps/create-app.md:43]: The same

File: content/apps/create-app/activate-dotnet.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+```dotnet [activate:dotNET]
```
Comment (tomglvng) [inline content/apps/create-app/activate-dotnet.md:1]: And remove this file

File: content/apps/create-app/activate-java.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+```java [activate:Java]
```
Comment (tomglvng) [inline content/apps/create-app/activate-java.md:1]: And this one also

File: content/apps/create-app/activate-nodejs.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+```js [activate:NodeJS]
+
+params // data from your request handling
+storage // your own memory system
+
+// Retrieve GET query params from your own framework / http handler
+const { pim_url: pimUrl } = params;
+
+// Retrive your app's Client ID with your own system
```
Comment (tomglvng) [inline content/apps/create-app/activate-nodejs.md:9]:
```suggestion
// Retrieve your app's Client ID with your own system
```

File: content/apps/create-app/activate-python.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+import secrets
+from urllib.parse import urljoin
+
+params # data from your request handling
+storage # your own memory system
+
+# Retrieve GET query params from your own framework / http handler
+pim_url: str = params.get('pim_url')
+
+# Retrive your app's Client ID with your own system
```
Comment (tomglvng) [inline content/apps/create-app/activate-python.md:12]:
```suggestion
# Retrieve your app's Client ID with your own system
```

File: content/apps/create-app/callback-dotnet.md [hunk 7]
```diff
@@ -0,0 +1,2 @@
+```dotnet [callback:dotNET]
```
Comment (tomglvng) [inline content/apps/create-app/callback-dotnet.md:1]: Remove

File: content/apps/create-app/callback-java.md [hunk 8]
```diff
@@ -0,0 +1,2 @@
+```java [callback:Java]
```
Comment (tomglvng) [inline content/apps/create-app/callback-java.md:1]: Also remove

File: content/apps/create-app/callback-nodejs.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
+```js [callback:NodeJS]
+
+params // data from your request handling
+storage // your own memory system
+
+// Retrieve GET query params from your own framework / http handler
+const { code, state } = params;
+
+// Retrive your app's Client ID with your own logic
```
Comment (tomglvng) [inline content/apps/create-app/callback-nodejs.md:9]:
```suggestion
// Retrieve your app's Client ID with your own logic
```

File: content/apps/create-app.md (hunk 1 above)
Comment (damienSD) [reply content/apps/create-app.md]: Done

File: content/apps/create-app.md (hunk 2 above)
Comment (damienSD) [reply content/apps/create-app.md]: Done

File: content/apps/create-app/activate-dotnet.md (hunk 3 above)
Comment (damienSD) [reply content/apps/create-app/activate-dotnet.md]: Done

File: content/apps/create-app/activate-java.md (hunk 4 above)
Comment (damienSD) [reply content/apps/create-app/activate-java.md]: Done

File: content/apps/create-app/activate-python.md (hunk 6 above)
Comment (damienSD) [reply content/apps/create-app/activate-python.md]: Done

File: content/apps/create-app/callback-dotnet.md (hunk 7 above)
Comment (damienSD) [reply content/apps/create-app/callback-dotnet.md]: Done

File: content/apps/create-app/callback-java.md (hunk 8 above)
Comment (damienSD) [reply content/apps/create-app/callback-java.md]: Done

File: content/apps/create-app/callback-nodejs.md (hunk 9 above)
Comment (damienSD) [reply content/apps/create-app/callback-nodejs.md]: Done

**Review by tomglvng — APPROVED** [review]

**Pull request merged**